packages/common/style/index.css

```diff
@@ -47,6 +48,22 @@
   background: var(--md-grey-400);
 }
 
+jp-button[title="Update all packages"][aria-disabled="false"] > svg {
```

Comment:
Hi Rosio, selecting the button based on the title seems like it could be a bit brittle, because if we change the title text then the CSS has to be updated as well.

What if instead you defined a dataset and used that to select the button based on its action? For example:

```
      <ToolbarButtonComponent
        icon={externalLinkIcon}
        tooltip="Update all packages"
        onClick={props.onUpdateAll}
        enabled={props.hasUpdate}
        dataset={{ action: 'update-all' }}
      />
```

Then in the CSS selector you could do:

```
jp-button[data-action="update-all"][aria-disabled="false"] > svg {
  color: var(--jp-accent-color0);
}
```

Although I'm not positive that the action in the dataset maps to the data-action, so we might need to test that by selecting in the DOM.

Reply:
Hi Dan! Thank you for your review! I can definitely update to use a dataset, the `ToolbarButtonComponent` [passes the dataset props](https://github.com/jupyterlab/jupyterlab/blob/b3094f1156ad67e23f91f0cc8cc676123c8eb448/packages/ui-components/src/components/toolbar.tsx#L873C7-L873C25) down but they are just missing the `data-` prefix. I'll make those updates shortly!